.github/workflows/playwright.yml

```
@@ -0,0 +1,65 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
```

There's no master branch, and this should probably run in the same cases as the main CI workflow:
```suggestion
  push:
  push:
    branches-ignore:
      - "dependabot/**"
      - "daily-test-sync"
  pull_request:
```

Arguably, this should actually just be a second job inside that main `ci.yml`...